cirq/contrib/noise_models/noise_models.py

```diff
@@ -173,16 +174,136 @@ def noisy_moment(self, moment: 'cirq.Moment',
             ]
 
 
-def simple_noise_from_calibration_metrics(calibration: engine.Calibration
+class PerQubitDepolarizingWithDampedReadoutNoiseModel(devices.NoiseModel):
+    """DepolarizingWithDampedReadoutNoiseModel which allows probabilities to be
+    specified separately for each qubit.
+
+    This simulates asymmetric readout error. The noise is structured
+    so the T1 decay is applied, then the readout bitflip, then measurement.
+    Note that T1 decay is only applied to measurement, not other gates.
+
+    In moments without measurement, all qubits affected by an operation will
+    have a depolarizing channel applied after the original operation. Qubits
+    that remain idle will be unaffected by this model.
+
+    As with the DepolarizingWithDampedReadoutNoiseModel, if a circuit contains
+    measurements, they must be in moments that don't also contain gates.
+    """
+
+    def __init__(
+            self,
+            depol_prob_map: Dict['cirq.Qid', float] = None,
+            bitflip_prob_map: Dict['cirq.Qid', float] = None,
+            decay_prob_map: Dict['cirq.Qid', float] = None,
+    ):
+        """A depolarizing noise model with damped readout error.
+
+        All error modes are specified on a per-qubit basis. To omit a given
+        error mode from the noise model, leave its map blank when initializing
+        this object.
+
+        Args:
+            depol_prob_map: Map of depolarizing probabilities for each qubit.
+            bitflip_prob: Probability of a bit-flip during measurement.
+            decay_prob: Probability of T1 decay during measurement.
+                Bitflip noise is applied first, then amplitude decay.
+        """
+        for prob_map, desc in [(depol_prob_map, "depolarization prob"),
+                               (bitflip_prob_map, "readout error prob"),
+                               (decay_prob_map, "readout decay prob")]:
+            if prob_map:
+                for qubit, prob in prob_map.items():
+                    value.validate_probability(prob, f'{desc} of {qubit}')
+        self.depol_prob_map = depol_prob_map
+        self.bitflip_prob_map = bitflip_prob_map
+        self.decay_prob_map = decay_prob_map
+
+    def noisy_moment(self, moment: 'cirq.Moment',
+                     system_qubits: Sequence['cirq.Qid']):
```

Comment:
Optional: add a short docstring?

Reply:
There's a docstring for the base NoiseModel class - is there anything here that you think needs additional explanation beyond that?

Reply:
That's why I left it as optional.  It's a method without a docstring but might be simple enough to leave off the docstring.  I'll leave it to your discretion whether to add any implementation details here.

Reply:
SGTM. I think the docstring for the class gives pretty good coverage of this, so I'll leave as-is.